src/decode.rs

```diff
@@ -2928,9 +2935,9 @@ unsafe fn decode_b_inner(
 
         // context updates
         if is_comp {
-            splat_tworef_mv(c, t, bs, b, bw4 as usize, bh4 as usize);
+            splat_tworef_mv(c, t, &mut f.rf.r, bs, b, bw4 as usize, bh4 as usize);
         } else {
-            splat_oneref_mv(c, t, bs, b, bw4 as usize, bh4 as usize);
+            splat_oneref_mv(c, t, &mut f.rf.r, bs, b, bw4 as usize, bh4 as usize);
```

Comment:
I can do that in a follow-up PR once `DisjointMut` lands.  I think it's good to separate that interior mutability change from the initial change to `Vec`s and indices here.

Reply:
I'm not sure we can land this without the interior mutability. It sounds like we don't want to add new mutable references to the data in `f`/`Rav1dFrameData` because Stephen's pending changes make all the references to `f` immutable. So unless we can land the followup PR right after landing this, merging this as-is would cause conflicts. @rinon can you confirm?

Reply:
Yes, this will interfere with the frame context mutability work. Let's not land this yet, please.